src/components/PopupWithConfirmation.js

```diff
@@ -8,8 +8,8 @@ export default class PopupWithConfirmation extends Popup {
     this._submitButtonText = this._submitButton.textContent;
   }
 
-  open() {
-    super.open();
+ {
+    super();
   }
 
```

Comment:
In `PopupWithConfirmation` you need to add text `"Deleting..."` instead of `Saving...` in line 20


```js

  setButtonText(submit, buttonText = "Deleting...") {
```